README.md

```diff
@@ -154,6 +154,30 @@ for peer <- [subscriber, publisher], do: Spell.close(peer)
 
 See `Spell.Role.Publisher` and `Spell.Role.Subscriber` for more information.
 
+#### Pattern-based subscriptions
+
+By default, peers subscribe to topics using an exact matching policy. Using
+pattern-based subscriptions a peer can receive messages that match certain
+criteria.
+
+```elixir
+# assuming we have a subscriber peer connected
+
+# exact match
+{:ok, subscription} = Spell.call_subscribe(subscriber, "com.spell.my_topic")
+# subscriber will receive messages published in the "com.spell.my_topic" topic
+
+# prefix match
+{:ok, subscription} = Spell.call_subscribe(subscriber, "com.spell.my_topic_prefix", options: %{match: :prefix})
```

Comment:
I am totally with you, I thin it would be better, I didn't want to change it in case it was like that for a reason. I'll give it a try